package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
     "eb-start": "npm run dist && APP_ENV=development node index",
     "dist": "NODE_ENV=production webpack --config webpack.config.js",
     "postinstall": "npm run dist",
-    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/*.test.js",
+    "test": "node_modules/.bin/mocha test/helpers/browser.js test/unit/$TEST_FILE",
```

Comment:
The file you are trying to test is not really an env variable though, so it's weird to think of it like that. I like that I can specify a file and if I don't, it runs all the tests.

My issue here is that this removes the development unit test service. So I have to run the command every time I update my tests. The way it was before, I ran `npm run test-file test/unit/Home.test.js` and if I updated any tests in `Home.test.js` the test runner would automatically update and test again. I prefer the previous way where I didn't have to start the runner every time.